generator/src/main/java/org/apache/kafka/message/MessageDataGenerator.java

```diff
@@ -1893,6 +1909,18 @@ private String fieldDefault(FieldSpec field) {
                 headerGenerator.addImport(MessageGenerator.UUID_CLASS);
                 return "UUID.fromString(\"" + field.defaultString() + "\")";
             }
+        } else if (field.type() instanceof FieldType.Float64FieldType) {
+            if (field.defaultString().isEmpty()) {
+                return "0D";
```

Comment:
Nit: I think `0.0` is idiomatic in Java.

Reply:
Done.